hibernate-core/src/main/java/org/hibernate/boot/internal/MetadataBuilderImpl.java

```diff
@@ -627,6 +628,12 @@ public MetadataBuildingOptionsImpl(StandardServiceRegistry serviceRegistry) {
 
 			this.multiTenancyStrategy =  MultiTenancyStrategy.determineMultiTenancyStrategy( configService.getSettings() );
 
+			this.xmlMetadataEnabled =  configService.getSetting(
+					AvailableSettings.XML_MAPPING_ENABLED,
```

Comment:
Minor nitpick: could we be consistent and either choose metadata or mapping? The attribute is `xmlMetadataEnabled` whereas the setting is `XML_MAPPING_ENABLED`.

Reply:
ok. which one do you want?

Reply:
I don't really care. Mapping makes more sense IMHO but it's the one requiring the most changes so not sure it's the right choice :).